torchvision/prototype/features/_feature.py

```diff
@@ -84,10 +86,10 @@ def __torch_function__(
         else:
             return output
 
-    def horizontal_flip(self) -> Any:
+    def horizontal_flip(self) -> _Feature:
```

Comment:
Nit: How about returning `T` where `T = TypeVar("T", bound=_Feature)`?

Reply:
I tried that https://github.com/pytorch/vision/pull/6331/commits/41a3f30fb24c264aa81de2d03e056c3ab79be2c2 with already defined `F` but mypy was failing...